Code/MinimalLib/common.h

```diff
@@ -1090,26 +1094,159 @@ std::string parse_inchi_options(const char *details_json) {
 }
 #endif
 
-struct LogHandle {
+namespace {
+#ifdef RDK_BUILD_THREADSAFE_SSS
+std::mutex &logger_mutex_get() {
+  // create on demand
```

Comment:
If this is C++11 or above you only need the logger_mutex_get, the static creation of _mutex will be thread-safe initialized and not require std::call_once

https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/n4594.pdf

From the section 6.7:

such a variable is initialized the first time control passes through its declaration; such a variable is considered initialized upon the completion of its initialization. [...] If control enters the declaration concurrently while the variable is being initialized, the concurrent execution shall wait for completion of the initialization.

Then there's a footnote:

The implementation must not introduce any deadlock around execution of the initializer.

Reply:
We have the same code elsewhere in RDKit (e.g., in `Code/GraphMol/DistGeomHelpers/Embedder.cpp` and `Code/GraphMol/MolPickler.cpp`); I would rather leave it as is here and then submit a separate PR to remove `std::call_once` if we believe that is unnecessary.

Reply:
Back then it was necessary, now it is not.  For new code we should use the current spec IMO

Reply:
As it makes code a lot simpler

Reply:
But the code is correct in both cases so I won’t fight this one

Reply:
@bp-kelley Hi Brian, I have applied your suggestion and removed `std::call_once` from this PR.
Are you OK to merge this PR now or do you need more time?

Reply:
@bp-kelley Gentle poke: I have applied your suggestion and removed std::call_once from this PR.
Are you OK to merge now or do you need more time?

Reply:
I admit to wishing there was an easier way to do this, but it finally makes sense to me.

LGTM